### PR TITLE
Mixin to override FOV multiplier instead of using the forge event

### PR DESCRIPTION
**Pull request by Caedis**
Confirmed working in the full pack

### CODE REVIEW

File: src/main/java/com/gtnewhorizons/angelica/mixins/early/angelica/misc/MixinEntityPlayerSP_DynamicFOV.java
Comment (mitchej123) [inline src/main/java/com/gtnewhorizons/angelica/mixins/early/angelica/misc/MixinEntityPlayerSP_DynamicFOV.java:14]: Please document why we're doing a mixin instead of using the event

**Review by mitchej123 — APPROVED** [review]
Approved pending documentation - disabled automerge

File: src/main/java/com/gtnewhorizons/angelica/proxy/ClientProxy.java
Comment (mitchej123) [inline src/main/java/com/gtnewhorizons/angelica/proxy/ClientProxy.java:279]: actually, try setting this to priority=EventPriority.LOWEST (last to execute)....

File: src/main/java/com/gtnewhorizons/angelica/mixins/early/angelica/misc/MixinEntityPlayerSP_DynamicFOV.java
Comment (Caedis) [reply src/main/java/com/gtnewhorizons/angelica/mixins/early/angelica/misc/MixinEntityPlayerSP_DynamicFOV.java]: Changed back to event based

**Comment by Caedis** [comment]
checking in the full pack before merging

**Comment by Caedis** [comment]
Confirmed working in full pack with the event priority set to lowest

**Pull request merged**
